cpp/CMakeLists.txt

```diff
@@ -37,28 +40,12 @@ if(${CMAKE_TOOLCHAIN_FILE} MATCHES "vcpkg.cmake$")
     set(ARCTICDB_USING_VCPKG ON)
 endif()
 
-if(NOT DEFINED ENV{CMAKE_BUILD_PARALLEL_LEVEL})
```

Comment:
Could you point me at where this is called now that you've refactored it out to a script?